packages/cms-cli/commands/customObject/schema/update.js

```diff
@@ -0,0 +1,69 @@
+const {
+  loadConfig,
+  validateConfig,
+  checkAndWarnGitInclusion,
+} = require('@hubspot/cms-lib');
+const { logger } = require('@hubspot/cms-lib/logger');
+const { logErrorInstance } = require('@hubspot/cms-lib/errorHandlers');
+const { getAbsoluteFilePath } = require('@hubspot/cms-lib/path');
+const { validatePortal, isFileValidJSON } = require('../../../lib/validation');
+const { trackCommandUsage } = require('../../../lib/usageTracking');
+const {
+  addTestingOptions,
+  setLogLevel,
+  getPortalId,
+} = require('../../../lib/commonOpts');
+const { ENVIRONMENTS } = require('@hubspot/cms-lib/lib/constants');
+const { getEnv } = require('@hubspot/cms-lib/lib/config');
+const { logDebugInfo } = require('../../../lib/debugInfo');
+const { updateSchema } = require('@hubspot/cms-lib/api/schema');
+
+exports.command = 'update <schemaObjectType> <definition>';
+exports.describe = 'Update an existing Custom Object Schema';
+
+exports.handler = async options => {
+  const { definition, schemaObjectType } = options;
+  setLogLevel(options);
+  logDebugInfo(options);
+  const { config: configPath } = options;
+  loadConfig(configPath);
+  checkAndWarnGitInclusion();
+
+  if (!(validateConfig() && (await validatePortal(options)))) {
+    process.exit(1);
+  }
+  const portalId = getPortalId(options);
+
+  trackCommandUsage('schema-update', null, portalId);
```

Comment:
I think this should be `custom-object-schema-update` and the same prefix should be used in the other commands so that the naming scheme is consistent with the other subcommands (e.g. `command-subcommand-subsubcommand`)